apps/explorer/src/app/components/blocks/home/blocks-table.tsx

```diff
@@ -12,31 +12,34 @@ interface BlocksProps {
 
 export const BlocksTable = ({ data, showTransactions }: BlocksProps) => {
   if (!data?.result) {
-    return <>No block data</>;
+    return <div className="mb-28">Awaiting block data</div>;
   }
 
   return (
     <Table>
       {data.result?.block_metas?.map((block, index) => {
         return (
           <React.Fragment key={index}>
-            <tr>
-              <td>
-                <Link to={`/blocks/${block.header?.height}`}>
+            <tr className="bg-neutral-850 border-b-4 border-b-black">
+              <td className="pl-4 py-2">
+                <Link
+                  to={`/blocks/${block.header?.height}`}
+                  className="text-vega-yellow"
+                >
                   {block.header?.height}
                 </Link>
               </td>
-              <td>
+              <td className="px-8 text-center">
```

Comment:
Lets avoid duplicating these classNames with a table cell component?

```
// in compoments/table/table.tsx
const TableCell = ({ children, className }) => {
  const cellClasses = classnames(
    'px-8', // default styles
    className
  )
  return <td className={cellClasses}>{children}</td>
}

// then use
<TableCell className="text-center">
```

Reply:
No problem. There's too much variety in the classes needed for defaults though, it's only a couple that use `px-8`, so I'd end up having to add extra classes to the other cells to undo the default. I'll make a component anyway and we can discuss whether to do more after.